src/main/resources/db/migration/0200/V249__ProjectVoting.sql

```diff
@@ -0,0 +1,27 @@
+-- Add vote options enum table (YES, CONDITIONAL, NO)
+CREATE TABLE accelerator.vote_options (
+    id INTEGER PRIMARY KEY GENERATED BY DEFAULT AS IDENTITY,
+    value TEXT NOT NULL,
+    UNIQUE (value)
+);
+
+-- Add whitelisted voters per project
+CREATE TABLE accelerator.project_voters (
+    user_id BIGINT NOT NULL REFERENCES users ON DELETE CASCADE,
+    project_id BIGINT NOT NULL REFERENCES projects ON DELETE CASCADE,
+    PRIMARY KEY (user_id, project_id)
+);
+
+-- Add voting records
+CREATE TABLE accelerator.project_vote_records (
+    user_id BIGINT NOT NULL REFERENCES users ON DELETE CASCADE,
+    project_id BIGINT NOT NULL REFERENCES projects ON DELETE CASCADE,
+    vote_option_id BIGINT NOT NULL REFERENCES accelerator.vote_options,
```

Comment:
A thought: if this were nullable, we wouldn't need a separate `project_voters` table. We'd populate this table with the roster of voters we would have inserted in that table.